docs.bruno/testing.bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: testing
+  type: http
+  seq: 1
+}
+
+get {
+  url: 
```

Comment:
It seems like the 'url' field in the 'get' block is left empty. Please ensure to provide a valid URL for the HTTP request. [important]